plugins/themes/controllers/files_controller.py

```diff
@@ -43,25 +43,39 @@ def __init__(self, app, handler):
             "/files/layers/delete/<string:layername>", "delete_layer",
             self.delete_layer, methods=["GET"]
         )
+        # Custom templates
+        # upload 
+        app.add_url_rule(
+            '/files/templates/upload', 'upload_template', self.upload_template,
+            methods=["GET", "POST"]
+        )
+        # delete
+        app.add_url_rule(
+            "/files/templates/delete/<string:templatename>", "delete_template",
+            self.delete_template, methods=["GET"]
+        )
 
         self.app = app
         self.handler = handler
         self.template_dir = "plugins/themes/templates"
 
         config_handler = handler()
         self.resources_path = config_handler.config().get("qgs_resources_path")
-
+        self.info_templates_path = config_handler.config().get("info_templates_path")
```

Comment:
Please document the setting in the schema definition [1].

[1] https://github.com/qwc-services/qwc-admin-gui/blob/master/schemas/qwc-admin-gui.json#L107

Reply:
And please also add the setting in the check here [1]

[1] https://github.com/qwc-services/qwc-admin-gui/blob/master/plugins/themes/__init__.py#L17

Reply:
Should be good now